File: app/src/main/java/com/loopeer/codereader/utils/ColorUtils.java
package com.loopeer.codereader.utils;

import android.content.Context;
import android.support.v4.content.ContextCompat;

public class ColorUtils {

    public static String getColorString(Context context, int res) {
        return String.format("#%06X"
                , 0xFFFFFF & ContextCompat.getColor(context
                        , res));
    }
}

File: code-reader-kt/src/main/java/com/loopeer/codereaderkt/utils/ColorUtils.kt
package com.loopeer.codereaderkt.utils

import android.content.Context
import android.support.v4.content.ContextCompat


object ColorUtils {

    fun getColorString(context: Context, res: Int): String {
        return String.format("#%06X", 0xFFFFFF and ContextCompat.getColor(context, res))
    }
}